core/migrations/0037_auto_20240924_1020.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 3.2.23 on 2024-09-24 08:20
```

Comment:
Why is this migration needed? 

field `elu_accession`  was a field of CoreTrackedModel class (parent class of CoreTrackedDBModel which is parent class to all updated classes in this migration)

Reply:
this migration was missing in the test instance. it was never pushed to the develop branch.